content/en/contribute/code/core/deploy-on-eks.md

```diff
@@ -233,13 +233,14 @@ And then follow these steps:
       * `secret` - this should match the version you cloned from
       * `user` - use `medic` user
       * `uuid` - this should match the version you cloned from
-      * `couchdb_node_storage_size` - use the same size as the volume you just cloned 
+      * `couchdb_node_storage_size` - use the same size as the volume you just cloned
+      * `account-id` - this should always be `720541322708`
       * `host` - this should be your username followed by `dev.medicmobile.org`.  For example `mrjones.dev.medicmobile.org`
       * `hosted_zone_id` - this should always be `Z3304WUAJTCM7P`
       * `preExistingDataAvailable` - set this to be `true`
+      * `dataPathOnDiskForCouchDB` - use the subPath you got in the step above. For example `storage/medic-core/couchdb/data`
       * `preExistingEBSVolumeID-1` - set this to be the ID from step 2. For example `vol-f9dsa0f9sad09f0dsa`
       * `preExistingEBSVolumeSize`  - use the same size as the volume you just cloned
-      *  `dataPathOnDiskForCouchDB` - use the subPath you got in the step above. For example `storage/medic-core/couchdb/data`
    
 7. Deploy this to development per the [steps above](#starting-and-stopping-aka-deleting). NB - **Be sure to call `kubectl config use-context arn:aws:eks:eu-west-2:720541322708:cluster/dev-cht-eks` before you call** `./cht-deploy`! Always create test instances on the dev cluster.
```

Comment:
Can we also include a section about how to delete the volume and snapshot? I feel like cleanup is important, and tracing dangling snapshots will be really difficult in the future.